clippy_lints/src/lib.rs

```diff
@@ -914,6 +917,7 @@ pub fn register_plugins(reg: &mut rustc_plugin::Registry<'_>, conf: &Conf) {
         let_if_seq::USELESS_LET_IF_SEQ,
         literal_representation::INCONSISTENT_DIGIT_GROUPING,
         literal_representation::UNREADABLE_LITERAL,
+        main_recursion::MAIN_RECURSION,
```

Comment:
Since this is a style lint, this should appear twice in `lib.rs`. Please run `./util/dev update_lints` again.